client/state/selectors/index.js

```diff
@@ -238,6 +239,7 @@ export isSiteAutomatedTransfer from './is-site-automated-transfer';
 export isSiteBlocked from './is-site-blocked';
 export isSiteOnFreePlan from './is-site-on-free-plan';
 export isSiteOnPaidPlan from './is-site-on-paid-plan';
+export isSiteSignupStore from './is-site-signup-store';
```

Comment:
Is `isSiteSignupStore` used for anything? I can't find any place where it would be called.

Reply:
Yeah. I'm actually thinking about getting rid of that selector completely. Maybe we'll even remove that site option if `has_pending_automated_transfer` works out well.